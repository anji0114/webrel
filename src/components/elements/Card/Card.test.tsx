import renderer from 'react-test-renderer';
import { Card } from '.';

test('Card snapshot', () => {
  const tree = renderer
    .create(<Card as='div'>カードコンポーネント</Card>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
