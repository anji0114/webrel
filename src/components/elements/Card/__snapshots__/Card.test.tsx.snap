// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot 1`] = `
<div
  className="p-6 rounded-lg bg-white border border-gray-300 shadow-sm"
>
  カードコンポーネント
</div>
`;
